app/(app)/vehicles/index.tsx

```diff
@@ -61,19 +61,23 @@ const VehiclesScreen = () => {
     closeModal()
   }
 
+  if (vehicles.length === 0) {
+    return (
+      <ScreenView title={t('title')} backgroundVariant="dots">
+        <NoVehicles />
+      </ScreenView>
```

Comment:
Meh, I don't like repeating the title like this, but I know we don't have better approach now. :D